### DISCUSSION TITLE
Glassmorphic design while waiting for April Release

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5670] [Show and tell]
Just with some little custom css.

<img width="1187" alt="Screenshot 2022-05-01 at 12 07 09 PM" src="https://user-images.githubusercontent.com/80512170/166139397-ac5d27f1-d8ca-4434-a337-648d6ae9c355.png">

**Comment by melohagan** [comment]
That looks really nice. Kind of has a modern Windows Aero vibe to it. 

It's a shame about those table pagination arrows. Should probably be made to have transparent backgrounds in the code base.

**Reply by poirazis** [reply]
Thank you @melohagan !

As more themeing capabilities come along , the more designs we will make :) There's no rush!
